### ISSUE TITLE
History menu does not insert items to the prompt when using with Oh My Zsh

### ISSUE THREAD

**Issue by mehanoid** [marlonrichert/zsh-autocomplete#101]
### Describe the bug
It seems that the history menu on the dev branch conflicts with Oh My Zsh or some of the default configurations that it provides. If I try to select some history item and insert it with ↩︎, then the menu just closes without any result. BTW multi-selection with ⇥ key works, but pressing ↩︎ afterwards would insert at least two commands at once, as expected though.

### To Reproduce

```shell
# *Always* test with the following three steps:
$ cd $(mktemp -d)  # Create a temp dir and enter it
$ ZDOTDIR=$PWD HOME=$PWD zsh -df  # Start a subshell without config files
% source path/to/oh-my-zsh.sh
% source path/to/zsh-autocomplete.plugin.zsh
% # press ↑ to show history menu
% # select menu item
% # press ↩︎
```

### Expected behavior

History item appears in the prompt

### Actual behavior

History item does not appear in the prompt

### Desktop
 - OS + distribution: Arch Linux 2020.09.25
 - Zsh version: 5.8
 - Plugin version: 654fa962713b30b3b85a5b82ae437cb162aa7dcc

### Additional context

- Oh My Zsh version: 93c837fec8e9fe61509b9dff9e909e84f7ebe32d

**Comment by mehanoid** [comment]
It seems that there is already open issue https://github.com/marlonrichert/zsh-autocomplete/issues/96, I'm sorry that I didn't notice it. But it is said to be fixed on the dev branch, and I still have this issue with that version.

**Comment by marlonrichert** [comment]
Should be fixed now on `master` with c69b60e665d0b510f63792319eec25f10a225eed.
